.github/workflows/mobile-traffic_director.yml

```
@@ -38,11 +42,12 @@ jobs:
     - name: 'Run GcpTrafficDirectorIntegrationTest'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GCP_SERVICE_ACCOUNT_JWT_TOKEN: ${{ secrets.GCP_SERVICE_ACCOUNT_JWT_TOKEN }}
+        # GCP_JWT_PRIVATE_KEY: ${{ secrets.GCP_SERVICE_ACCOUNT_JWT_TOKEN }}
+        GCP_JWT_PRIVATE_KEY: nirvana
+        ENVOY_IP_TEST_VERSIONS: v4only
       run: |
-        cd mobile && ./bazelw test \
+        cd mobile && ./bazelw run \
```

```suggestion
        cd mobile
        ./bazelw run
```

im going to go through the others to cleanup these - but atm these are using `&&` and `\` and it keeps causing issues